### PR TITLE
GCW-3345 - Fix validation issue for user info on account details page

### PR DESCRIPTION
**Pull request by saunmanoj888**
### Ticket Link: 

https://jira.crossroads.org.hk/browse/GCW-3345

### What does this PR do?

Fix validation issue for user full name in accounts details page for browse app

### CODE REVIEW

File: app/controllers/account_details.js [hunk 1]
```diff
@@ -25,6 +25,13 @@ export default Controller.extend({
   preferredContactNumber: "",
   mobilePhone: "",
   isMobileApp: config.cordova.enabled,
+  userInfoError: "",
+  firstName: computed("model.user.firstName", function() {
+    return $.trim(this.get("model.user.firstName"));
```
Comment (swatijadhav) [inline app/controllers/account_details.js:30]: Don't go jQuery way, use js.

```suggestion
    return this.get("model.user.firstName").trim();
```

File: app/controllers/account_details.js [hunk 2]
```diff
@@ -25,6 +25,13 @@ export default Controller.extend({
...
+  }),
+  lastName: computed("model.user.lastName", function() {
+    return $.trim(this.get("model.user.lastName"));
```
Comment (swatijadhav) [inline app/controllers/account_details.js:33]:
```suggestion
    return this.get("model.user.lastName").trim();
```

File: app/templates/account_details.hbs [hunk 3]
```diff
@@ -62,13 +62,13 @@
                         </div>
                         <div class="small-5 columns">
                           <label class="form-label title-label">{{t "account.first"}}</label>
-                          {{input type="text" value=model.user.firstName id="firstName" classNames="number_input center_align_placeholder" required='true'}}
+                          {{input type="text" value=model.user.firstName id="firstName" classNames="number_input center_align_placeholder" required='true' focus-out="validateUserInfo"}}
                         </div>
                         <div class="small-5 columns">
                           <label class="form-label title-label">{{t "account.last"}}</label>
-                          {{input type="text" value=model.user.lastName id="lastName" classNames="number_input center_align_placeholder" required='true'}}
+                          {{input type="text" value=model.user.lastName id="lastName" classNames="number_input center_align_placeholder" required='true' focus-out="validateUserInfo"}}
```
Comment (shreyas098) [inline app/templates/account_details.hbs:69]: validateUserInfo is a action , hence needs to be called accessed in the particular way
```suggestion
                          {{input type="text" value=model.user.lastName id="lastName" classNames="number_input center_align_placeholder" required='true' focusOut=(action "validateUserInfo")}}
```

File: app/templates/account_details.hbs [hunk 4]
```diff
@@ -62,13 +62,13 @@
                         </div>
                         <div class="small-5 columns">
                           <label class="form-label title-label">{{t "account.first"}}</label>
-                          {{input type="text" value=model.user.firstName id="firstName" classNames="number_input center_align_placeholder" required='true'}}
+                          {{input type="text" value=model.user.firstName id="firstName" classNames="number_input center_align_placeholder" required='true' focus-out="validateUserInfo"}}
```
Comment (shreyas098) [inline app/templates/account_details.hbs:65]: validateUserInfo is a action , hence needs to be called accessed in the ember closure way
```suggestion
                          {{input type="text" value=model.user.lastName id="lastName" classNames="number_input center_align_placeholder" required='true' focusOut=(action "validateUserInfo")}}
```

File: app/controllers/account_details.js [hunk 5]
```diff
@@ -149,6 +156,10 @@ export default Controller.extend({
 
   actions: {
     saveAccount() {
+      if (this.get("inValidUserName")) {
```
Comment (shreyas098) [inline app/controllers/account_details.js:159]: This "if" check is not required , please remove it

**Review by shreyas098 — APPROVED** [review]

**Review by swatijadhav — APPROVED** [review]

**Pull request merged**